lib/CustomFields/pages/EditCustomFieldsRecord/tests/EditCustomFieldsRecord-test.js

```diff
@@ -20,7 +20,7 @@ describe('EditCustomFieldsRecord', () => {
 
   const renderComponent = (props = {}) => {
     const FormComponent = () => (
-      <Form onSubmit={() => {}}>
+      <Form onSubmit={() => { }}>

```

Comment:
Is this to fix a eslint error or something?

Reply:
@BogdanDenis, no, I did it by mistake